tools/notification-configuration/notification-creator/NotificationConfigurator.cs

```diff
@@ -186,7 +186,16 @@ private async Task SyncTeamWithCodeOwnerFile(BuildDefinition pipeline, WebApiTea
 
                 // Get contents of CODEOWNERS
                 logger.LogInformation("Fetching CODEOWNERS file");
-                var managementUrl = new Uri(pipeline.Repository.Properties["manageUrl"]);
+                var managementUrl = default(Uri);
+                if (pipeline.Repository.Properties.ContainsKey("manageUrl"))
```

Comment:
Do we need to check `String.IsNullOrEmpty` on the value here as well? Otherwise `new Uri()` will throw. Similarly, I think this code can be slightly simplified into just an if/else block instead of doing a null check below, since it will never be null if we hit this first if block.

Reply:
Change to other code